.github/workflows/ci.yml

```diff
@@ -238,6 +238,11 @@ jobs:
         run: docker-compose -f cirq-rigetti/docker-compose.test.yaml up -d
       - name: Coverage check
         run: check/pytest-and-incremental-coverage -n auto --rigetti-integration
+      - name: Upload coverage reports to Codecov
+        uses: codecov/codecov-action@v3
+        env:
+          # Admin required to access repo settings > secrets and variable > actions
+          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
I think we can take this out, the token is not needed for public repositories - https://github.com/codecov/codecov-action#usage.